### PR TITLE
improved local node.js installation instructions

### PR DESCRIPTION
**Pull request by keydon**
+ also added systax highlighting for listings
+ simplified image references

This closes OmerTu/GoogleHomeKodi#108

### CODE REVIEW

**Comment by keydon** [comment]
mhh lets better delay my pull-request until we get those other two almost rdy pull requests through :)

**Comment by keydon** [comment]
now is up to date with the recent changes

File: README.md [hunk 1]
```diff
@@ -170,20 +197,79 @@ SyslogIdentifier=nodejs-example
...
+   docker stop googlehomekodi
+   docker rm sinedied/googlehomekodi
+   ```
+</details>
+
+### **C) [OPTIONAL] controlling multiple kodi instances**
+There are two ways of implpementing the scenario of multiple kodi instance (i.e one in the living room and another in the bedroom).
+1. Having two IFTTT-Applets per command with different phrases. Each applet is responsible for one instance. (i.e. the phrase `living room kodi play...` could target `https://mylivingroom.glitch.com/playpause` and the phrase `bedroom kodi play...` could target `https://bedroom.glitch.com/playpause`).
+2. When running a local node server you can utilize the kodi-hosts.config.js configuration.
+This configuration will allow you to address multiple kodi installations (e.g. "Hey Google, kodi play [movie name] in the bedroom") from a single IFTTT-Applet.
```
[the remark this answers is not in the document]
Comment (p0psicles) [reply README.md]: Your right it's still two applets. Only now they have the have the same URL endpoint, but different payload.

**Comment by keydon** [comment]
fixed you comments.

and for the multiple instances, i wrote something completly diffrent than i wanted to ;O
Now also rewritten. I left out the part that simple commands could be implemented with a single applet, it just confuses ppl.

**Comment by OmerTu** [comment]
Looks great, thanks!

**Pull request merged**